### DISCUSSION TITLE
WARN: ITS A VIRUS!

### DISCUSSION THREAD

**Discussion by tpglitch** [Dadoum/Sideloader#54] [General]
I tried to install and it asks for a password!

**Comment by Dadoum** [comment]
you could look at the code and compile it yourself yk, we're on github, this is not your average could-be-malware sideloader app.

What you probably encountered is that the default command launched by sideloader asks for the password. What you probably wanted to do is to add `-h` to the command.

**Comment by Dadoum** [comment]
also you can create an apple id specifically for sideloading too (and you probably should have done that anyway, to avoid having to go through 2FA)
